tests/Functional/Fixtures/Symfony/AppCache.php

```diff
@@ -25,4 +29,14 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
 
         return $response;
     }
+
+    /**
+     * Made public to allow event subscribers to do refresh operations.
+     *
+     * {@inheritDoc}
+     */
+    public function fetch(Request $request, $catch = false)
```

Comment:
so actually it does change visibility but there seems to be some bug with interface vs trait somewhere: http://stackoverflow.com/questions/31877844/php-trait-exposing-a-method-and-interfaces